src/components/SkipLink.tsx

```diff
@@ -9,6 +9,7 @@ const StyledSkipLink = styled(Link)`
   align-items: center;
   height: 3rem;
   background: ${({ theme }) => theme.palette.section.black};
+  clip: rect(1px, 1px, 1px, 1px);
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/CSS/clip#syntax
Note: Where possible, authors are encouraged to use the newer clip-path property instead.

Reply:
Brukes ikke clip kun på bilder??

Reply:
`The clip CSS property defines a visible portion of an element`, så funker tydeligvis på alle elementer, men bilder er nok det mest naturlige. uutilsynet har samme på sidene sine. overrasket over hvor knotete det var å få til noe som bare skulle vises ved fokus 🤔